run_config_input.py

```diff
@@ -384,6 +384,21 @@ def run_normal_config_file(config_file="./examples/configuration.json"):
     with open("./logs/" + model_with_config_name + "/config.json", "w") as f:
         json.dump(config, f)
 
+    if (
+        "continue" in config["NeuralNetwork"]["Training"]
```

Comment:
Is this our only optional argument? Just curious

Reply:
> Is this our only optional argument? Just curious

No, we have at least one more, i.e., "denormalize_output". By default it is "False".